### DISCUSSION TITLE
merging type systems failed: unable to build object definition

### DISCUSSION THREAD

**Discussion by kokutas** [99designs/gqlgen#1855] [Q&A]
### What happened?
merging type systems failed: unable to build object definition: unable to find type: github.com/99designs/gqlgen/graphql/introspection.Schema

### versions
 - `gqlgen version`? 
 - v0.14.0-dev
 - `go version`?
 - go version go1.16.10 darwin/amd64
 - dep or go modules?
 - 	github.com/99designs/gqlgen v0.15.1

**Comment by ipfans** [comment]
Can you attach a demo to reproduce this issue?

**Comment by JUSTICEESSIELP** [comment]
I just solved it  so its a matter of using both the instropection and a particular gqlgen version 
Let me explain follow these steps 






****************************************************


mkdir gqlgen-todos
cd gqlgen-todos
go mod init github.com/[username]/gqlgen-todos



printf '// +build tools\npackage tools\nimport (_ "github.com/99designs/gqlgen"\n _ "github.com/99designs/gqlgen/graphql/introspection")' | gofmt > tools.go



go get -d github.com/99designs/gqlgen@v0.17.20                  
go get github.com/99designs/gqlgen/graphql/introspection

 go run github.com/99designs/gqlgen init




*******************************************************


I tried so many ways but this worked

**Reply by coolwednesday** [reply]
Hey, so even I am facing the same issue. The above hack is also not working.
